ethereum/gateway-eth-ts/src/utils/gas.test.ts

```diff
@@ -1,7 +1,9 @@
 import { estimateGasPrice, getDefaultOracle, DEFAULT_GAS_PRICES, GasPrices } from './gas';
 import assert = require('assert');
+const { should } = require('chai');
 import { utils } from 'ethers/lib/ethers';
 import { GasPriceOracle } from 'gas-price-oracle';
+should();
```

Comment:
is this line correct?

Reply:
It's required to access should assertions on typescript tests

Reply:
ah ok, we normally use chai with sinon